File: LeetCode/LeetCode_Easy/C#/IsAnagram.cs
class IsAnagram
{
    public bool Solution(string s, string t)
    {
        char[] first = s.ToCharArray();
        char[] second = t.ToCharArray();

        Array.Sort(first);
        Array.Sort(second);

        if (first.SequenceEqual(second))
        {
            return true;
        }

        return false;
    }
}

File: LeetCode/LeetCode_Easy/C++/IsAnagram.cpp
class Solution {
public:
    string sortString(string &str) {
        sort(str.begin(), str.end());
        return str;
    }

    bool isAnagram(string s, string t) {
        string first = sortString(s);
        string second = sortString(t);

        if (first == second)
            return true;

        return false;
    }
};